src/view/com/util/UserBanner.tsx

```diff
@@ -90,14 +93,11 @@ export function UserBanner({
 
   // setUserBanner is only passed as prop on the EditProfile component
   return onSelectNewBanner ? (
-    <EventStopper onKeyDown={false}>
+    <EventStopper onKeyDown={true}>
```

Comment:
Not sure we even need this anymore

Reply:
oh cool! I'll follow up with a PR to remove those, wanna test thoroughly first so won't do it here